modules/library/metadata/pom.xml

```diff
@@ -135,6 +135,12 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
+      <dependency>
+          <groupId>org.geotools.ogc</groupId>
+          <artifactId>net.opengis.ows</artifactId>
+          <version>23-SNAPSHOT</version>
```

Comment:
This should be the parameter ${project.version}

Reply:
Will do